website/dashboard/test_plots.js

```diff
@@ -6,14 +6,14 @@ document.addEventListener('DOMContentLoaded', function() {
                 var reference_values = data.map(d => d[parameter]);
                 var fit_values = data.map(d => d[parameter + '_fit']);
                 var errors = fit_values.map((d, i) => d - reference_values[i]);
-
-                var tolerance = reference_values.map((d, i) => data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d);
-                var negative_tolerance = tolerance.map(t => -t);
-
                 var minRefValue = Math.min(...reference_values);
                 var maxRefValue = Math.max(...reference_values);
                 // Create a range for tolerance trace x axis.
                 var xRange = [minRefValue, maxRefValue];
+
+                var tolerance = xRange.map((d, i) => data[reference_values.indexOf(d)]['atol'][parameter] + data[i]['rtol'][parameter] * d);
```

Comment:
Your index in `rtol` should be the same as in `atol`. You're choosing different datasets for both parts of the equation.

Reply:
yeah, that's how the plots look after correctly accessing the tolerances of the min and max reference values:

<img width="345" alt="image" src="https://github.com/user-attachments/assets/963bbd52-0313-4379-8af9-b40fb6f1858e">